docs/spark-structured-streaming.md

```diff
@@ -53,32 +50,36 @@ Iceberg only supports reading data from append snapshots. Overwrite snapshots ca
 To write values from streaming query to Iceberg table, use `DataStreamWriter`:
 
 ```scala
-val tableIdentifier: String = ...
 data.writeStream
     .format("iceberg")
     .outputMode("append")
     .trigger(Trigger.ProcessingTime(1, TimeUnit.MINUTES))
-    .option("path", tableIdentifier)
     .option("checkpointLocation", checkpointPath)
-    .start()
+    .toTable("database.table_name")
```

Comment:
Which versions of Flink does this work for? Is it new and that's why we didn't show it before? May want a note about compatibility if that's the case.

Reply:
Ah, that's a good call. Since 3.1: https://github.com/apache/spark/blob/e4114f67e12a235b4784fcbfa6f6ba9b44a5e715/sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala#L267